docs/tasks/administer_cluster_quotas.md

```diff
@@ -210,6 +210,171 @@ To create the ClusterQueue, run the following command:
 kubectl apply -f cluster-total.yaml
 ```
 
+## Multiple ClusterQueues and borrowing cohorts
+
+Two or more ClusterQueues can borrow unused quota from other ClusterQueues in
+the same cohort.
+
+Using the following example, you can establish a cohort `team-ab` that includes
+ClusterQueues `team-a-cq` and `team-b-cq`.
+
+```yaml
+# team-a-cq.yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ClusterQueue
+metadata:
+  name: team-a-cq
+spec:
+  namespaceSelector: {}
+  cohort: team-ab
+  requestableResources:
+  - name: "cpu"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 9
+        ceiling: 15
+  - name: "memory"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 36Gi
+        ceiling: 60Gi
+```
+
+```yaml
+# team-b-cq.yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ClusterQueue
+metadata:
+  name: team-b-cq
+spec:
+  namespaceSelector: {}
+  cohort: team-ab
+  requestableResources:
+  - name: "cpu"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 12
+  - name: "memory"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 48Gi
+```
+
+Note that the ClusterQueue `team-a-cq` also defines [ceilings](/docs/concepts/cluster_queue.md#ceilings).
+This restricts the ability of the ClusterQueue to borrow the unused quota from
+the cohort up to the configured ceiling, even if the quota is completely unused.
+
+To create these ClusterQueues, save the preceding manifests and run the
+following command:
+
+```shell
+kubectl apply -f team-a-cq.yaml -f team-b-cq.yaml
+```
+
+## Multiple ClusterQueue with dedicated and fallback flavors
```

Comment:
One thing worth mentioning is that in the case of multiple flavors, borrowing will happen first before falling down to the next flavor even if quota was available for the next flavor

Reply:
Instead, I decided to add a section in ClusterQueue concept for borrowing semantics